### ISSUE TITLE
Activation Code Error

### ISSUE THREAD

**Issue by songursamet** [alexal1/Insomniac#414]
My bot gives that error anyone can help me, please? I was using the same interact.jason 4-5 months ago.

```
Traceback (most recent call last):
  File "C:\Users\samet\Desktop\Insomniac\insomniac\action_get_my_profile_info.py", line 9, in get_my_profile_info
    profile_view = TabBarView(device).navigate_to_profile()
  File "C:\Users\samet\Desktop\Insomniac\insomniac\views.py", line 140, in navigate_to_profile
    self.navigate_to(TabBarTabs.PROFILE)
  File "C:\Users\samet\Desktop\Insomniac\insomniac\views.py", line 203, in navigate_to
    raise LanguageNotEnglishException()
insomniac.views.LanguageNotEnglishException

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\samet\Desktop\Insomniac\insomniac\session.py", line 313, in run
    self.prepare_session_state(args, device_wrapper, app_version, save_profile_info=True)
  File "C:\Users\samet\Desktop\Insomniac\insomniac\session.py", line 267, in prepare_session_state
    self.session_state.my_following_count = get_my_profile_info(device, self.username)
  File "C:\Users\samet\Desktop\Insomniac\insomniac\action_get_my_profile_info.py", line 27, in get_my_profile_info
    print(COLOR_FAIL + describe_exception(e) + COLOR_ENDC)
  File "C:\Users\samet\Desktop\Insomniac\insomniac\utils.py", line 370, in describe_exception
    trace = ''.join(traceback.format_exception(etype=type(ex), value=ex, tb=ex.__traceback__)) if with_stacktrace else ''
TypeError: format_exception() got an unexpected keyword argument 'etype'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\samet\Desktop\Insomniac\start.py", line 4, in <module>
    insomniac.run(activation_code)
  File "C:\Users\samet\Desktop\Insomniac\insomniac\__init__.py", line 33, in run
    insomniac_session.run()
  File "C:\Users\samet\Desktop\Insomniac\insomniac\session.py", line 345, in run
    print(COLOR_FAIL + describe_exception(ex) + COLOR_ENDC)
  File "C:\Users\samet\Desktop\Insomniac\insomniac\utils.py", line 370, in describe_exception
    trace = ''.join(traceback.format_exception(etype=type(ex), value=ex, tb=ex.__traceback__)) if with_stacktrace else ''
TypeError: format_exception() got an unexpected keyword argument 'etype'
```

**Comment by emre-aktas** [comment]
Hello mate, if you have an activation code, can you write to me via mail or telegram? I would like to ask a few questions. Thank you for attention.

magma017@gmail.com
@emredeveloper

**Comment by songursamet** [comment]
> Hello mate, if you have an activation code, can you write to me via mail or telegram? I would like to ask a few questions. Thank you for attention.
> 
> [magma017@gmail.com](mailto:magma017@gmail.com) @emredeveloper

I'm sorry I don't have an activation code. I was using this bot for free a few months ago but I'm not sure if it works still or not.

**Comment by dpw1** [comment]
I am experiencing the same issue. Just to make it clear, to use it I'll need to purchase an activation code regardless? I thought it was only for the additional features.

**Comment by lambonas** [comment]
Anyone found a fix for this one ?

**Comment by dpw1** [comment]
Not really. If the app is meant to be used only for paid users that is ok, but it would be nice to add it to the readme.

**Comment by lambonas** [comment]
> Not really. If the app is meant to be used only for paid users that is ok, but it would be nice to add it to the readme.

its not, I've activated and still get this error dude.

**Comment by lambonas** [comment]
@dpw1 the fix is here. https://github.com/alexal1/Insomniac/pull/415 thanks to @gtcdevop all is working well on my end, open up the device_dacade.py and edit the is alive code to server alive.

**Comment by alexal1** [comment]
Error was on latest Python version only, fixed in v3.8.1
